src/rgw/rgw_admin.cc

```diff
@@ -558,8 +557,10 @@ static int get_cmd(const char *cmd, const char *prev_cmd, const char *prev_prev_
     if (strcmp(cmd, "list") == 0)
       return OPT_BI_LIST;
   } else if (strcmp(prev_cmd, "period") == 0) {
-    if (strcmp(cmd, "prepare") == 0)
-      return OPT_PERIOD_PREPARE;
+    if (strcmp(cmd, "prepare") == 0) {
+      cerr <<"command is obsolete; please check usage and/or man page" <<std::endl;
+      return 0;
+    }
```

Comment:
i appreciate you wanting to document the deprecation, but the reason i don't think it's necessary is that we've never documented this command (outside of `radosgw-admin --help`) as part of the [multisite configuration process](http://docs.ceph.com/docs/master/radosgw/multisite/)